OpenAPI/decentralized-iso20022-v1.yaml

```diff
@@ -1,152 +1,395 @@
-  - name: External
-    description: For a financial institution to receive a message from another financial institution.
+  - name: Internal/External
+    description: '<strong>External:</strong> Recieve ISO 20022 messages from another financial institution. <br><br> <strong>Internal:</strong> Send ISO 20022 messages to another financial institution.'
 
 paths:
   /api/v1/pacs:
     post:
       tags:
-        - Internal
-      summary: Send a pacs message to another financial institution.
-      description: The endpoint where the financial insitution tells the microservice to send a pacs message to another financial institution.<br><br>A valid pacs message should be sent with a header head.001.001.02.
-      requestBody:
-        $ref: '#/components/requestBodies/Pacs.008.001.09Body'
-      responses:
-        '200':
-          description: A pacs.002.001.12 message with the header head.001.001.02.
-          content:
-            application/xml:
-              schema: 
-                type: array
-                items: 
-                  type: string
-        '400':
-          description: An error message indicating that the message was invalid.
-          content:
-            application/xml:
-              schema: 
-                type: array
-                items: 
-                  type: string
-        '403':
-          description: An error message indicating that the sender or the receiver could not be authenticated.
-          content:
-            application/xml:
-              schema: 
-                type: array
-                items: 
-                  type: string
-        '500':
-          description: An error message indicating that an unknown error occurred.
-          content:
-            application/xml:
-              schema: 
-                type: array
-                items: 
-                  type: string
-        '502':
-          description: An error message indicating that the message could not be sent.
-          content:
-            application/xml:
-              schema: 
-                type: array
-                items: 
-                  type: string
-
-  /api/v1/pacs:
-    post:
-      tags:
-        - External
+        - Internal/External
       summary: Receive a pacs message from another financial institution.
-      description: The endpoint from which the microservice receives pacs messages from another financial institution.<br><br>A valid pacs message should be received with a header head.001.001.02.
+      description: <strong>Internal:</strong> The endpoint where the financial institution sends the pacs.008.001.09 message to the microservice for validation before forwarding it to the recieving financial institution specified in the 'to' element in the 'AppHdr' element. <br><br><strong>External:</strong> The endpoint from which the microservice receives pacs messages from another financial institution. A valid pacs message should be received with a header head.001.001.02.
```

Comment:
I noticed that the term was used before too, but I think it should be changed to "communication service" since that is what it is referred to as in a bunch of other places.

Reply:
Sounds good, will change that!